svir/test/integration/test_drive_oq_engine.py

```diff
@@ -55,8 +57,17 @@ def setUp(self):
         self.session = Session()
         self.hostname = os.environ.get('OQ_ENGINE_HOST',
                                        'http://localhost:8800')
+        self.reset_gui()
+
+    def tearDown(self):
+        del self.session
+        del self.hostname
+        del self.viewer_dock
+
```

Comment:
Is this the part solving the segfault? If so, maybe a comment is in order.

Reply:
The segfault is solved by another thing. I've just added a comment